genesis/camino_genesis_test.go

```diff
@@ -0,0 +1,290 @@
+					thisConfig.Camino.Allocations = append(thisConfig.Camino.Allocations, KopernikusConfig.Camino.Allocations...)
+					return &thisConfig
+				}(),
+			},
+			err: errors.New("staker ins't consortium member"),
+		},
+		"consortium member not kyc verified": {
+			args: args{
+				config: func() *Config {
+					thisConfig := KopernikusConfig
+					thisConfig.Camino.Allocations = []CaminoAllocation{{AddressStates: AddressStates{ConsortiumMember: true}, PlatformAllocations: []PlatformAllocation{{NodeID: ids.GenerateTestNodeID()}}}}
+					thisConfig.Camino.Allocations = append(thisConfig.Camino.Allocations, KopernikusConfig.Camino.Allocations...)
+					return &thisConfig
+				}(),
+			},
+			err: errors.New("consortium member not kyc verified"),
+		},
+		"wrong validator duration": {
+			args: args{
+				config: func() *Config {
+					thisConfig := KopernikusConfig
+					thisConfig.Camino.Allocations = []CaminoAllocation{{AddressStates: AddressStates{ConsortiumMember: true, KYCVerified: true}, PlatformAllocations: []PlatformAllocation{{NodeID: ids.GenerateTestNodeID(), ValidatorDuration: 0}}}}
+					thisConfig.Camino.Allocations = append(thisConfig.Camino.Allocations, KopernikusConfig.Camino.Allocations...)
+					return &thisConfig
+				}(),
+			},
+			err: errors.New("wrong validator duration"),
+		},
+		"repeated staker allocation": {
+			args: args{
+				config: func() *Config {
+					thisConfig := KopernikusConfig
+					thisConfig.Camino.Allocations = []CaminoAllocation{{AddressStates: AddressStates{ConsortiumMember: true, KYCVerified: true}, PlatformAllocations: []PlatformAllocation{{NodeID: KopernikusConfig.Camino.Allocations[0].PlatformAllocations[0].NodeID, ValidatorDuration: 1}}}}
+					thisConfig.Camino.Allocations = append(thisConfig.Camino.Allocations, KopernikusConfig.Camino.Allocations...)
+					return &thisConfig
+				}(),
+			},
+			err: errors.New("repeated staker allocation"),
+		},
+		"no staker allocations": {
+			args: args{
+				config: func() *Config {
+					thisConfig := KopernikusConfig
+					thisConfig.Camino.Allocations = []CaminoAllocation{{PlatformAllocations: []PlatformAllocation{}}}
+					return &thisConfig
+				}(),
+			},
+			err: errors.New("no staker allocations"),
+		},
+		"valid": {
+			args: args{
+				config: &KopernikusConfig,
+			},
+		},
+	}
+	for name, tt := range tests {
+		t.Run(name, func(t *testing.T) {
+			err := validateCaminoConfig(tt.args.config)
+
+			if tt.err != nil {
+				require.ErrorContains(t, err, tt.err.Error())
+				return
+			}
+			require.NoError(t, err)
+		})
+	}
+}
+
+func TestBuildCaminoGenesis(t *testing.T) {
+	ctx := snow.DefaultContextTest()
+	rewards := reward.NewCalculator(defaultRewardConfig)
+	ctx.NetworkID = constants.KopernikusID
+	baseDB := memdb.New()
+	db := versiondb.New(baseDB)
+	cfg := defaultConfig()
+
+	addrs := set.Set[ids.ShortID]{}
+	_, _, avaxAddrBytes, _ := address.Parse(initialAdmin)
```

Comment:
imo, its better to not ignore errors in test, even if we expect them to not happen - better to rely on `require.NoError(t, err)`

Reply:
done